configs/ghactions/deploy_github_pages.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
This new script overlaps alot with the [deploy script for travis](https://github.com/reposense/publish-RepoSense/blob/master/configs/travis/deploy_github_pages.sh), with the difference only in the name of variables. 

Perhaps can just use a single deploy script, and standardising the naming of the variables between travis and GitHub action runs?

Reply:
Opps, forgot to login to my main account before commenting.

Reply:
I have now merged the two deploy scripts into one and factored out the different variables.